puffin_egui/src/lib.rs

```diff
@@ -104,6 +104,7 @@ use std::{
     fmt::Write as _,
     sync::{Arc, Mutex},
 };
+use time::{macros::format_description, OffsetDateTime};
```

Comment:
You can move the `use` statement to unside `fn format_time`

* less pollution of file namespace
* when reading `format_time` it's easier to find where the names come from